### PR TITLE
Added Network Console to experimental features

### PR DESCRIPTION
**Pull request by erdraud**
I also filed a PR for adding the screenshot I added to the Network Console section: https://github.com/MicrosoftDocs/edge-developer/pull/808

### CODE REVIEW

**Comment by zoherghadyali** [comment]
To address @brendyna's feedback on What's New for Edge 85, I think the CORS limitations of Network Console should be called out here, not in What's New. @erdraud please include that as well.

File: microsoft-edge/devtools-guide-chromium/experimental-features.md [hunk 1]
```diff
@@ -45,18 +45,19 @@ The following sections describe the new experimental features that are available
...
 | [Enable webhint](#enable-webhint) | 85 or later | 
+| [Enable Network Console](#enable-network-console) | 85 or later |
 
 ### Enable custom keyboard shortcuts settings tab
 
 Provides a new **Shortcuts** page in [DevTools Settings][DevToolsCustomizeSettings] that enables matching [keyboard shortcuts][DevToolsShortcuts] in the DevTools to [VS Code][VisualstudioCode].  
 
-Once you have enabled this experiment, open [DevTools Settings][DevToolsCustomizeSettings] again by pressing `Shift`+`?`.  Navigate to the new **Shortcuts** page.  Select **DevTools (Default)** in the **Match shortcuts from preset** dropdown and select **Visual Studio Code**.  The keyboard shortcuts in the DevTools now match the shortcuts for equivalent actions in VS Code.  
+Once you have enabled the experiment, open [DevTools Settings][DevToolsCustomizeSettings] again by select `Shift`+`?`.  Navigate to the new **Shortcuts** page.  Select **DevTools (Default)** in the **Match shortcuts from preset** dropdown and select **Visual Studio Code**.  The keyboard shortcuts in the DevTools now match the shortcuts for equivalent actions in VS Code.
```
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/experimental-features.md:54]:
```suggestion
Once you have enabled the experiment, open [DevTools Settings][DevToolsCustomizeSettings] again using select `Shift`+`?`.  Navigate to the new **Shortcuts** page.  Select **DevTools (Default)** in the **Match shortcuts from preset** dropdown and select **Visual Studio Code**.  The keyboard shortcuts in the DevTools now match the shortcuts for equivalent actions in VS Code.  
```

File: microsoft-edge/devtools-guide-chromium/experimental-features.md [hunk 2]
```diff
@@ -80,14 +81,26 @@ Normally, tools such as **Elements** and **Network** may only be opened in the m
...
 
 :::image type="complex" source="./media/experiments-webhint.png" alt-text="webhint feedback in the Issues panel" lightbox="./media/experiments-webhint.png":::
    webhint feedback in the Issues panel  
 :::image-end:::      
 
 <!--Available in Microsoft Edge version 85 and later.  -->  
 
+### Enable Network Console
+
+**Network Console** is the working title for an experimental component for Chromium-based DevTools for making synthetic network requests over HTTP.  You may use the **Network Console** experiment during the dev-test loop for web APIs.  The **Network Console** experiment is not feature-complete during the initial release.  Early availability during the dev-test loop enables you to receive rapid iterations, experiment in the open, and see your ideas for the **Network Console** implemented quickly.
```
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/experimental-features.md:94]:
```suggestion
**Network Console** is the working title of an experiment to make synthetic network requests over HTTP.  You may use the **Network Console** experiment to call web APIs.  
```

File: microsoft-edge/devtools-guide-chromium/experimental-features.md [hunk 3]
```diff
@@ -80,14 +81,26 @@ Normally, tools such as **Elements** and **Network** may only be opened in the m
...
    webhint feedback in the Issues panel  
 :::image-end:::      
 
 <!--Available in Microsoft Edge version 85 and later.  -->  
 
+### Enable Network Console
+
+**Network Console** is the working title for an experimental component for Chromium-based DevTools for making synthetic network requests over HTTP.  You may use the **Network Console** experiment during the dev-test loop for web APIs.  The **Network Console** experiment is not feature-complete during the initial release.  Early availability during the dev-test loop enables you to receive rapid iterations, experiment in the open, and see your ideas for the **Network Console** implemented quickly.  
+
+Once you enable the experiment, restart your DevTools.  Open the **Network** pane, hover on the network request you are interested in changing and sending again, open the contextual menu \(right-click\), and choose **Edit and Replay**.  The **Network Console** opens and your network request information is pre-loaded for you to then edit and send.
```
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/experimental-features.md:96]:
```suggestion
After enabling the experiment, ensure you restart the DevTools. To use the Network Console:
1.  Open the **Network** pane.
1.  Find the network request that you want to change and resend.
1.  Open the contextual menu \(right-click\), and choose **Edit and Replay**. 
1.  When the **Network Console** opens, edit the network request information.
1.  Select **Send**.  
```

File: microsoft-edge/devtools-guide-chromium/experimental-features.md (hunk 2 above)
Comment (erdraud) [reply microsoft-edge/devtools-guide-chromium/experimental-features.md:94]: #sign-off

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**